lib/oneview-sdk/resource/api200/volume_template.rb

```diff
@@ -33,23 +33,19 @@ def initialize(client, params = {}, api_ver = nil)
 
       # Create the resource on OneView using the current data
       # @note Calls refresh method to set additional data
+      # @param [Hash] header The header options for the request (key-value pairs)
       # @raise [OneviewSDK::IncompleteResource] if the client is not set
       # @raise [StandardError] if the resource creation fails
       # @return [Resource] self
-      def create
-        ensure_client
-        response = @client.rest_post(self.class::BASE_URI, { 'Accept-Language' => 'en_US', 'body' => @data }, @api_version)
-        body = @client.response_handler(response)
-        set_all(body)
+      def create(header = DEFAULT_REQUEST_HEADER)
+        super(header.merge('Accept-Language' => 'en_US'))
```

Comment:
Isn't it worth it to add `'Accept-Language' => 'en_US'` to `def create(header = DEFAULT_REQUEST_HEADER)`?
As in, `def create(header = {'Accept-Language' => 'en_US'})`